src/gap_filler/Shak scratch file.py

```diff
@@ -0,0 +1,11 @@
+
+#variables - the task
+
+sentence_1 = ("I am in the park.")
```

Comment:
Well I think you've cheated a little lol you have given it a tuple which contains a string.

However in the rest of the code our language model produces a list of words. So in order for this code to work with the rest of the codebase you need to make it work on a list of strings.

So for example to rewrite your `sentence_1` as a list of strings it should look like this:`sentence_1 = ["I", "am", "in", "the", "park"]`

Then you have to LOOP over the items in that list and when one of them is the work 'in' replace it with '__'.

Reply:
I struggle to create a string with` sentence_1 = ["I", "am", "in", "the", "park"]` due to receiving an error explaining a string can only have 3 arguments and this sentence has 5. 

Understood, let me work on this again.

Reply:
I'm not quite sure what you mean, you don't need to create a string like that, you need a variable that is a list of strings. Because our language model takes the string and converts it into a list similar to this.

Reply:
Hey @alex3142 I am trying here, and have sent an image into the whatsapp group. 
I have researched loops and do not seem to be able to successfully  use the replace function. Can you please provide a working example?